### PR TITLE
Deployment/venus lens

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR add the new testnet deployment of the new version of VenusLens contract.

Resolves #202 

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by chechu — APPROVED** [review]

**Review by defcon022 — APPROVED** [review]
lgtm

File: hardhat.config.ts [hunk 1]
```diff
@@ -16,6 +16,16 @@ require("dotenv").config();
 const BSCSCAN_API_KEY = process.env.BSCSCAN_API_KEY;
 const DEPLOYER_PRIVATE_KEY = process.env.DEPLOYER_PRIVATE_KEY;
 
+// This is a sample Hardhat task. To learn how to create your own go to
```
Comment (coreyar) [inline hardhat.config.ts:19]: We can remove this comment

File: deployments/bsctestnet/VenusLens.json [hunk 2]
```diff
@@ -0,0 +1,1015 @@
+{
+  "address": "0x11c8dC3DcA87E8205ec01e6d79Be9442d31701d3",
```
Comment (coreyar) [inline deployments/bsctestnet/VenusLens.json:2]: Since this is a new deployment we should also bump the package version

Comment (0xlucian) [reply deployments/bsctestnet/VenusLens.json:2]: yeah bumped it

**Pull request merged**